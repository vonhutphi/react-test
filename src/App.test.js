import Tabs from "./components/tabs";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Tabs />).toJSON();
  expect(tree).toMatchSnapshot();
});

