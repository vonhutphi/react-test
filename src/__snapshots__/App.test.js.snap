// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="tab-pane-wrapper"
  >
    <ul
      className="tab-bar"
    >
      <li
        className="active"
        onClick={[Function]}
      >
        Top Games
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        New Games
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Slots
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Jackpots
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Live
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Blackjack
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Roulette
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Table
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Poker
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Other
      </li>
    </ul>
    <div
      className="tab-bar-mobile"
    >
      <div
        className="tab-bar-mobile-wrapper"
      >
        <div
          className="head-menu-wrapper"
        >
          <div
            className="head-menu"
          >
            <div
              className="selected-tab"
            >
              <span>
                Top Games
              </span>
            </div>
            <div
              className="menu-icon"
              onClick={[Function]}
            >
              <div
                className="line line-1"
              />
              <div
                className="line line-2"
              />
              <div
                className="line line-3"
              />
            </div>
          </div>
        </div>
        <div
          className="dropdown-wrapper false"
        >
          <ul
            className="dropdown-list"
          >
            <li
              className="active"
              onClick={[Function]}
            >
              <span>
                Top Games
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                New Games
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Slots
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Jackpots
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Live
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Blackjack
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Roulette
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Table
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Poker
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Other
              </span>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="tab-content-wrapper"
  >
    <div
      className="tab-content row"
    />
  </div>
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="wrapper"
>
  <div
    className="tab-pane-wrapper"
  >
    <ul
      className="tab-bar"
    >
      <li
        className="active"
        onClick={[Function]}
      >
        Top Games
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        New Games
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Slots
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Jackpots
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Live
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Blackjack
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Roulette
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Table
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Poker
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Other
      </li>
    </ul>
    <div
      className="tab-bar-mobile"
    >
      <div
        className="tab-bar-mobile-wrapper"
      >
        <div
          className="head-menu-wrapper"
        >
          <div
            className="head-menu"
          >
            <div
              className="selected-tab"
            >
              <span>
                Top Games
              </span>
            </div>
            <div
              className="menu-icon"
              onClick={[Function]}
            >
              <div
                className="line line-1"
              />
              <div
                className="line line-2"
              />
              <div
                className="line line-3"
              />
            </div>
          </div>
        </div>
        <div
          className="dropdown-wrapper false"
        >
          <ul
            className="dropdown-list"
          >
            <li
              className="active"
              onClick={[Function]}
            >
              <span>
                Top Games
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                New Games
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Slots
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Jackpots
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Live
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Blackjack
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Roulette
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Table
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Poker
              </span>
            </li>
            <li
              className=""
              onClick={[Function]}
            >
              <span>
                Other
              </span>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="tab-content-wrapper"
  >
    <div
      className="tab-content row"
    />
  </div>
</div>
`;
